src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,18 +45,24 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={(event) => onQueryChange(event.target.value)}
       />
       <span className="icon is-left">
         <i className="fas fa-magnifying-glass" />
       </span>
 
       <span className="icon is-right" style={{ pointerEvents: 'all' }}>
```

Comment:
Using inline styles is a bad practice, use css instead.

Reply:
that's part of the initial code, but i have fixed it, thank you)